Description: replace "itemx" with "item" to make new makeinfo happy

--- cgdb-0.6.7.orig/doc/cgdb.texinfo
+++ cgdb-0.6.7/doc/cgdb.texinfo
@@ -686,7 +686,7 @@ Searches wrap around the end of file.  T
 @itemx :continue
 Send a continue command to GDB.
 
-@itemx :down
+@item :down
 Send a down command to GDB.
 
 @item :e
@@ -752,7 +752,7 @@ Send a step command to GDB.
 @item :syntax
 Turn the syntax on or off.
 
-@itemx :up
+@item :up
 Send an up command to GDB.
 
 @item :map @var{lhs} @var{rhs}
